crates/boundless-cli/src/commands/povw/claim.rs

```diff
@@ -87,6 +88,10 @@ pub struct PovwClaim {
 
     #[clap(flatten, next_help_heading = "Prover")]
     prover_config: ProverConfig,
+
+    /// The RPC URL to use to submit the reward claim transaction.
+    #[clap(long, env = "ETH_RPC_URL")]
```

Comment:
nit: maybe rename it to `REWARD_RPC_URL` so in case it's decided to use a different chain for the reward (for example base due to tx fees in eth), it's a no brainer :)